libs/ui-lib/lib/ocm/components/clusterConfiguration/operators/MtvOperatorCheckbox.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { FormGroup, HelperText, HelperTextItem, Tooltip } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+import {
+  getFieldId,
+  PopoverIcon,
+  MTV_LINK,
+  OperatorsValues,
+  ClusterOperatorProps,
+} from '../../../../common';
+import { OcmCheckboxField } from '../../ui/OcmFormFields';
+import { useNewFeatureSupportLevel } from '../../../../common/components/newFeatureSupportLevels';
+import { useFormikContext } from 'formik';
+import MtvRequirements from './MtvRequirements';
+
+const Mtv_FIELD_NAME = 'useMigrationToolkitforVirtualization';
+
+const MtvLabel = ({
+  disabledReason,
+  clusterId,
+}: {
+  disabledReason?: string;
+  clusterId: string;
+}) => (
+  <>
+    <Tooltip hidden={!disabledReason} content={disabledReason}>
+      <span>Install Migration Toolkit for Virtualization </span>
+    </Tooltip>
+    <PopoverIcon
+      component={'a'}
+      headerContent="Additional requirements"
+      bodyContent={<MtvRequirements clusterId={clusterId} />}
+    />
+  </>
+);
+
+const MtvHelperText = () => {
+  return (
+    <HelperText>
+      <HelperTextItem variant="indeterminate">
+        The Migration Toolkit for Virtualization (MTV) enables you to migrate virtual machines from
+        VMware vSphere, Red Hat Virtualization, or OpenStack to OpenShift Virtualization running on
+        Red Hat OpenShift.{' '}
+        <a href={MTV_LINK} target="_blank" rel="noopener noreferrer">
+          {'Learn more'} <ExternalLinkAltIcon />
+        </a>
+      </HelperTextItem>
+    </HelperText>
+  );
+};
+
+const MtvCheckbox = ({ clusterId }: { clusterId: ClusterOperatorProps['clusterId'] }) => {
+  const featureSupportLevelContext = useNewFeatureSupportLevel();
+  const { values } = useFormikContext<OperatorsValues>();
+  const fieldId = getFieldId(Mtv_FIELD_NAME, 'input');
+  const [disabledReason, setDisabledReason] = useState<string | undefined>();
+
+  React.useEffect(() => {
+    const disabledReason = featureSupportLevelContext.getFeatureDisabledReason('MTV');
```

Comment:
This will currently always return `undefined` because

1. There is no `MTV` flag in the feature support levels from the BE. I assume it will get added in the near future?
2. The `MTV` case is missing from the `getNewFeatureDisabledReason` callback in [featureStateUtils.ts](https://github.com/openshift-assisted/assisted-installer-ui/blob/master/libs/ui-lib/lib/ocm/components/featureSupportLevels/featureStateUtils.ts#L156).

Reply:
@jgyselov yes, this will be added in the feature.
Added the MTV case to `getNewFeatureDisabledReason`